src/osd/OSDMap.cc

```diff
@@ -5198,7 +5199,9 @@ void OSDMap::get_random_up_osds_by_subtree(int n,     // whoami
     return;
   vector<int> subtrees;
   crush->get_subtree_of_type(subtree_type, &subtrees);
-  std::random_shuffle(subtrees.begin(), subtrees.end());
+  std::random_device rd;
+  std::default_random_engine rng{rd()};
+  std::shuffle(subtrees.begin(), subbtrees.end(), rng);
```

Comment:
s/subbtrees/subtrees/

Reply:
Grr, I fixed that.....
Repushed, see if that helps.